Makefile

```diff
@@ -0,0 +1,35 @@
+.DEFAULT_GOAL := all
+
+.PHONY: build64 build32 all clean install
+
+PKG_CONFIG_PATH_32 ?= /usr/lib32/pkgconfig
```

Comment:
Is this true for most systems? Might need to try to source this information from within meson, but I didn't find a good way.

Reply:
It's not. It would be `/usr/lib/i386-linux-gnu/pkgconfig` for Debian/Ubuntu and `/usr/lib/pkgconfig` for Fedora and openSUSE. But since vkBasalt doesn't actually link with vulkan libs, it should compile anyway, even in case of incorrect path here.

Reply:
@gasinvein I agree it's not a big deal especially since the `Makefile` is really just a helper at this point, but what's the standard meson solution for gleaning that information (if there is one)?

Reply:
AFAIK meson has no means to guess pkgconfig paths for foreign arches. But some distros provide separate pkgconfig commands for cross-compiling (e.g. `i686-pc-linux-gnu-pkg-config`), which you can specify in meson's cross-file `[binaries]` section.